### ISSUE TITLE
Error in audio, Elementary Os

### ISSUE THREAD

**Issue by MichiWoo** [elementary/switchboard-plug-sound#148]
Error: Cannot find audio devices, does not have audio, in windows I have a driver with Realtek High Definition Audio Driver, in elementary OS I have Linux 5.3.0-51-generic. If you could help me, thanks.

**Comment by NikkoUrza** [comment]
lo lograste solucionar?

**Comment by MichiWoo** [comment]
No, parece es la tarjeta de audio de este equipo, con otros sistemas
operativos linux pasa lo mismo. En otros equipos funciona perfecto
elementary os. Gracias

**Comment by NikkoUrza** [comment]
si ya instale otras versiones y pasa igual, estoy frustrado.
